site_root/input/ImplementationGuide-KidsFirst.json

```diff
@@ -574,6 +574,38 @@
         "reference": {
           "reference": "Organization/sc-001"
         }
+      },
+      {
+        "description": "Kids First StructureDefinition study, Base: ResearchStudy",
+        "exampleBoolean": false,
+        "name": "Kids First StructureDefinition/study",
+        "reference": {
+          "reference": "StructureDefinition/study"
+        }
+      },
+      {
+        "description": "Kids First StructureDefinition kfdrc-sequencing-experiment, Base: Task",
+        "exampleBoolean": false,
+        "name": "Kids First StructureDefinition/kfdrc-sequencing-experiment",
+        "reference": {
+          "reference": "StructureDefinition/kfdrc-sequencing-experiment"
+        }
+      },
+      {
+        "description": "Kids First Task se-001, Profiles: kfdrc-sequencing-experiment",
+        "exampleBoolean": true,
+        "name": "Kids First Task/se-001",
+        "reference": {
+          "reference": "Task/se-001"
+        }
+      },
+      {
+        "description": "Kids First Observation se-001",
```

Comment:
`"description": "Kids First Observation se-001"` should be `"description": "Kids First Observation se-001 Profiles: Observation"`. This is because this example resource doesn't have `meta.profile`. Please see the comment right below.